src/components/view/ContactInfo/contactInfoEventsCard.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+import { Card, CardHeader, CardText } from 'material-ui/Card';
+import { List, ListItem } from 'material-ui/List';
+
+import { getImageForEnv } from 'static/images/index'
+
+class ContactInfoEventsCard extends Component {
+
+    formatUidToDate = uid => {
```

Comment:
Why not just `formatUidToDate (uid) {` ?

#question

Reply:
bc es6 mannnnnn

.. I dunno, I'll change it